### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by olafbanki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The format is a single ZIP archive that bundles various delimited text files des
...
+The COLDP format was developed to overcome limitations existing in currently used formats for sharing taxonomic information, namely [Darwin Core Archives](https://dwc.tdwg.org/text/) and the Catalogue of Life submission format also known as [ACEF](http://www.catalogueoflife.org/content/contributing-your-data#ACEF) (Annual Checklist Exchange Format). Darwin Core Archives and ACEF could still be used for exchanging data to and from the Catalogue of Life ChecklistBank, but the COLDP format will support the most features. The following table provides an overview of different features supported in each of the 3 formats:
```
Comment (mdoering) [inline README.md:12]: I did not use COLDP on purpose as it sounds like COLD-P. Can we please stick with ColDP? It's been circulated at TDWG, in presentations, its in our code and separates COL from DataPackage much cleaner

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
-# CoL Data Package (COLDP) Specification
-The recommended exchange format for data to and from the CoL Clearinghouse
+# COL Data Package (COLDP) Specification
+The recommended exchange format for data to and from the COL ChecklistBank
```
Comment (mdoering) [inline README.md:3]: I am no native speaker, but when changing from clearinghouse to checklistbank I would also remove the `the` and just call it _from COL ChecklistBank_

Comment (olafbanki) [reply README.md:3]: It is ok to drop 'the'

File: README.md (hunk 1 above)
Comment (olafbanki) [reply README.md:12]: Fine, leave it.

**Pull request merged**